labgrid/remote/client.py

```diff
@@ -809,6 +823,21 @@ def create():
     loop.run_until_complete(ready.wait())
     return session[0]
 
+def find_role_by_place(config, place):
+    for role, role_config in config.items():
+        resources, _ = target_factory.normalize_role(role, role_config)
+        remote_places = resources.get('RemotePlace', {})
+        remote_place = remote_places.get(place)
+        if remote_place:
+            return role
+
+def find_any_role_with_place(config):
+    for role, role_config in config.items():
```

Comment:
As above, iterating over the role is not necessary.

Reply:
We still need the loop, but we don't need to pass the role name to normalize_config.